website/docs/reference/warehouse-profiles/rockset-profile.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Rockset Profile"
+---
+
+:::info Vendor-supported plugin
+
+Certain core functionality may vary. If you would like to report a bug, request a feature, or contribute, you can check out the linked repository and open an issue.
+
+:::
+
+## Overview of dbt-rockset
+
+**Maintained by:** Rockset, Inc.      
+**Source:** https://github.com/rockset/dbt-rockset
+**Core version:** v0.0.3 and newer    
```

Comment:
Could you update this to 0.19.2 and newer? I'm pretty sure this isn't supported with v0.0.3 :)